pkgs/applications/misc/djvulibre/default.nix

```diff
@@ -13,10 +13,10 @@ stdenv.mkDerivation rec {
   buildInputs = [ libjpeg libtiff librsvg libiconv ];
```

Comment:
While at it, please also split name into pname and version.

Reply:
I wanted to do code changes after adopting a package, to be "by maintainer".
Done.